athena/athena/endpoints.py

```diff
@@ -270,7 +270,9 @@ def feedback_provider(func: Union[
     Callable[[E, S], List[F]],
     Callable[[E, S], Coroutine[Any, Any, List[F]]],
     Callable[[E, S, C], List[F]],
-    Callable[[E, S, C], Coroutine[Any, Any, List[F]]]
+    Callable[[E, S, C], List[F]],
+    Callable[[E, S, C], Coroutine[Any, Any, List[F]]],
+    Callable[[E, S, G, C], List[F]],
```

Comment:
`Callable[[E, S, C], List[F]],` is there twice

```suggestion
    Callable[[E, S, C], Coroutine[Any, Any, List[F]]],
    Callable[[E, S, G, C], List[F]],
```